src/plugins/Network.js

```diff
@@ -37,6 +37,14 @@ module.exports = class Network extends OverlayPlugin {
         this.renderToolbarButton();
         this.renderWidget();
 
+        /*
+         * Redis message for enabling/disabling mobile throttling and 5G support
+         * could be sent without rendering the widget from srach
+         * to avoid recreation of widget elements, check this parameters before.
+         */
+        this.mobileThrottlingManaged = false;
+        this.network5Gmanaged = false;
```

Comment:
Maybe like this, as it seems a clearer wording to me:
```suggestion
        this.mobileThrottlingConfigured = false;
        this.network5GConfigured = false;
```